src/ca65/expr.c

```diff
@@ -417,6 +418,33 @@ static ExprNode* FuncDefined (void)
 
 
 
+static ExprNode* FuncDefinedMacro (void)
+/* Handle the .DEFINEDMACRO builtin function */
+{
+    Macro* Mac = 0;
+
+    /* Check for a macro or an instruction depending on UbiquitousIdents */
+
+    if (CurTok.Tok == TOK_IDENT) {
+        if (UbiquitousIdents) {
+            /* Macros CAN be instructions, so check for them first */
+            Mac = FindMacro (&CurTok.SVal);
+        } else {
+            /* Macros and symbols may NOT use the names of instructions */
+            if (FindInstruction (&CurTok.SVal) < 0) {
```

Comment:
You are probably right. I was thinking about that actually. I'll verify with some testing.